datalad/__init__.py

```diff
@@ -35,6 +35,20 @@
 else:
     ssh_manager = None
 
+try:
+    # this will fix the rendering of ANSI escape sequences
+    # for colored terminal output on windows
+    # it will do nothing on any other platform, hence it
+    # is safe to call unconditionally
+    import colorama
+    colorama.init()
+    atexit.register(colorama.deinit)
+except ImportError:
+    if on_windows:
+        lgr.warning(
+            "'colorama' Python module missing, terminal output may look garbled")
```

Comment:
It is just an assumption... Import could fail for other reason (bug in module etc)... That is why I usually prefer just to include exception detail in the message